spanner/session.go

```diff
@@ -906,7 +906,11 @@ func (p *sessionPool) createMultiplexedSession() {
 				p.mu.Lock()
 				p.multiplexedSessionCreationError = err
 				p.mu.Unlock()
-				p.mayGetMultiplexedSession <- true
+				select {
+				case p.mayGetMultiplexedSession <- true:
+				case <-c.ctx.Done():
+					continue
```

Comment:
nit: Do we need this line at all? The first line after this block is also just `continue`

Reply:
Not needed, removed